packaging/rpm-ostree.spec.in

```diff
@@ -237,6 +237,11 @@ $PYTHON autofiles.py > files.devel \
 # Setup rpm-ostree-countme.timer according to presets
 %post
 %systemd_post rpm-ostree-countme.timer
+# Only enable on rpm-ostree based systems and manually force unit enablement to
+# explicitly ignore presets for this security fix
+if [ -e /run/ostree-booted ]; then
```

Comment:
@travier When composing a tree on a non ostree system (my CI) this will not run
in my post script:
```
[ -e /run/ostree-booted ] || exit 42
```
running rpm-ostree compose (2023.7)
```
...
Executing postprocessing script
+ '[' -e /run/ostree-booted ']'
+ exit 42
error: Executing postprocessing script: bwrap(/usr/bin/rpmostree-treefile-postprocess-script): Child process killed by signal 42
```

Reply:
How are you composing your image? Are you adding that as a postscript to your manifest?

Reply:
Ignore my previous message, `/run/ostree-booted` is indeed present when running rpm post scripts, just not present when running `postprocess-script` (but not really needed)